wpunit.suite.yml

```diff
@@ -0,0 +1,24 @@
+# Codeception Test Suite Configuration
+
+# Suite for unit or integration tests that require WordPress functions and classes.
+
+# The wpRootFolder needs to be set to your wordpress directory.
+
+actor: WpunitTester
+modules:
+    enabled:
+        - WPLoader
+        - \Helper\Wpunit
+    config:
+        WPLoader:
+            wpRootFolder: "./public/"
+            dbName: "wpTests"
+            dbHost: "localhost"
+            dbUser: "wpTests"
+            dbPassword: "wpTests"
+            tablePrefix: "wp_"
+            domain: "localhost:8000"
```

Comment:
Does the fact that port here differs from port in `codeception.yml` and `tests/wpunit.suite.yml` above matter?

Is there a single parent config setting that can be set somewhere so the extra ones can be removed?